### PR TITLE
feat(dashboard): Whats new modal

### PR DESCRIPTION
**Pull request by tristandubbeld**
image

Learn more links will be addressed with a later linear task.

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      )}
+    </Media>
+  );
+};
+
+interface TopicProps {
+  children: React.ReactNode[];
+  icon: React.ComponentProps<typeof Icon>['name'];
+  // ❗️ TODO: Link
+  linkTo?: string;
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx:131]: XTD-226

File: packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                <Grid columnGap={5}>
+                  <Column span={matchSmall || matchMedium ? 6 : 3}>
+                    <Topic icon="repository">
+                      Projects is now <Text weight="700">Repositories</Text>: an
+                      improved git workflow powered by the cloud.
+                    </Topic>
+                  </Column>
+
+                  <Column span={matchSmall || matchMedium ? 6 : 3}>
+                    <Topic icon="dashboard">
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx:97]: This isn't the house icon, but we don't use it yet in the client codebase. Is this OK for the dashboard icon?

**Review by alexnm — APPROVED** [review]

File: packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx (hunk 2 above)
Comment (tristandubbeld) [reply packages/app/src/app/pages/Dashboard/Components/WhatsNew/WhatsNew.tsx]: Fixed it and we're now using the right icon.

**Review by olarclara — APPROVED** [review]
LGTM!

**Pull request merged**